### DISCUSSION TITLE
팀원 소개

### DISCUSSION THREAD

**Discussion by JeongJae-Private** [Pseudo-Lab/9th-Mastering-Kaggle#5] [Announcements]
자기소개 부탁 드립니다!

**Comment by JeongJae-Private** [comment]
![image](https://github.com/user-attachments/assets/14b215d9-0adc-48e1-86a0-250de767f0f5)


[성정재] ([링크드인](https://www.linkedin.com/in/정재-성-04b6a478?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base_contact_details%3BBTA6eTWATJmXDVA4yBSXBg%3D%3D))
프로젝트명 : 캐글 정복하기

### **인삿말**

안녕하세요! Data Scientist 로 일하고 있는 성정재 입니다.

### **간단한 이력**

학부 졸업 후 기구개발 직무에서 일을 하다, Data Scientist 직무로 옮겨 일을 하고 있습니다. 

### **현재 관심사**

RL, LLM 등 다양한 기술들에 관심이 많습니다 

### **특별한 경험**

삼성 그룹에서는 1~2년간 현지 언어와 문화를 익히도록 지원하는 해외 연수 프로그램을 운영하고 있는데요 2015년 중국 지역전문가로 뽑히게 되어 1년간 상해에서 생활을 하고 온 것이 가장 기억에 남습니다.

### **현재 하고 있는 것**

Product의 Sensor Data를 활용하여 문제점 원인 분석 및 예측 모델을 개발하고 있습니다.

### **취미**

책읽는 것과 사람들과 이야기 하는 것을 좋아합니다 😊

**Comment by Ujae22** [comment]
![image](https://github.com/user-attachments/assets/9a3d0d25-43e5-4bbb-a15c-19b2c9f94fd8)


[최우재]
프로젝트명 : 캐글 정복하기

### 인삿말
안녕하세요! 최우재라고 합니다. 학부 과정에서 산업공학을 전공하였고, 지금은 회사와 병행하여 데이터사이언스 대학원에서 석사 과정으로 재학하고 있습니다.

### 간단한 이력
학부 졸업 후 LG에너지솔루션 빅데이터분석팀에서 
일하고 있습니다.

### 현재 관심사
ML/DL

### 특별한 경험
유럽 여행을 갔다가 핸드폰을 소매치기를 당했는데 경찰에 신고한다고 했더니 돌려받은 경험이 있습니다.

### 현재 하고 있는 것
공정에서 발생하는 불량을 ML 모델로 탐지하는 과제에 투입되어 일하고 있습니다.

### 취미
사람들과 이야기 하기, 자전거 타고 양재천을 도는 것이 취미입니다.
(최근에 보드게임에도 입문했습니다~!)

**Comment by sungwu** [comment]
[조성우]
프로젝트명 : 캐글 정복하기

### 인삿말
안녕하세요! ML, DL은 잘 모르지만 열심히 참여해서 작지만 확실한 결과를 내보고 싶습니다. 잘 부탁드립니다! 

### 간단한 이력
로봇회사에서 자율주행 소프트웨어를 개발하고 있습니다. 

### 현재 관심사
시뮬레이션, 자동화, 강화학습에 관심을 가지고 있습니다. 

### 특별한 경험

개발에 참여한 로봇이 본격 서비스를 시작했습니다. 떨립니다.. 

### 현재 하고 있는 것

이동로봇의 경로 계획, 행동 계획 등을 개발하고 있습니다. 

### 취미
책 읽기, 유튜브 보기

**Comment by soyunshin** [comment]
[신소윤] 
프로젝트명 : 캐글 정복하기

## 인삿말
안녕하세요! 
데이터 분석가 신소윤 입니다.

## 간단한 이력
게임 회사의 이상 탐지팀에서 데이터 분석 업무를 하고 있습니다.

## 현재 관심사
다양한 데이터 분석 기법을 활용한 Kaggle 데이터 분석에 관심이 많습니다.

## 특별한 경험
롯데 월드에서 근무하며, 2년간 다양한 사람들을 만난 경험이 가장 기억에 남습니다.

## 현재 하고 있는 것
게임에서 발생하는 유저 행동 로그를 통해 이상 탐지 모델을 개발하고 있습니다.

## 취미
게임 하는 것과 유튜브 보는 것을 좋아합니다 😊

**Comment by Angela-Park-JE** [comment]
[박정은] ([링크드인](https://www.linkedin.com/in/angelapje)) 
프로젝트명 : 캐글 정복하기 

### 인삿말

안녕하세요! 데이터 분석가로 취업 준비중인 박정은입니다😁! 

### 간단한 이력

의류학, 문예창작 학사전공 후 취업 준비중 데이터 과학에 빠져서 공부하게 되었습니다. 

### 현재 관심사

이상치 탐지, 전달력 좋은 분석과 시각화에 관심이 많습니다. 
많이 알지는 못하지만,  AI와 관련된 문제와 규제 등에 관심이 있습니다. 

### 특별한 경험

작년에 야구 직관을 처음 가봤고, 올해에는 좋아하는 게임의 월드챔피언십 경기 직관을 다녀왔습니다. 
사람들과 열정을 함께 나눌 수 있는 공간이 특별했던 것 같습니다. 

### 현재 하고 있는 것

파이토치 튜토리얼 번역 컨트리뷰션에 참여하고 있습니다. 마침 컨트리뷰션 동기분들과 사이드 프로젝트로 [Kaggle에 있는 시계열 데이터](https://www.kaggle.com/competitions/store-sales-time-series-forecasting/overview) 예측 프로젝트를 진행하고 있습니다. 

### 취미

음악감상, 게임, 독서, 운동, 길에서 본 동물들 속으로 귀여워하기입니다😊.

**Comment by JeongSuJin** [comment]
![profile_pic_1.jpg](https://github.com/JeongSuJin/practice/blob/master/profile_pic_1.jpg?raw=true)

[정수진](https://www.linkedin.com/in/수진-정-bab273b7)
프로젝트명 : 캐글 정복하기

### **인삿말**
안녕하세요! 정수진입니다. 직장인용 데이터사이언스 대학원을 갓 졸업한 인공지능 병아리입니다. 

### **간단한 이력**
삼성전자에서 소프트웨어 엔지니어로 일하고 있습니다.

### **현재 관심사**
자연어처리, LLM, RAG 같은 기술들에 관심이 있습니다.

### **특별한 경험**
오래전에 대학생때 아프리카에서 렌트카로 사하라 사막을 가서 1박을 하고 새벽밤하늘을 구경했던게 기억에 남습니다.

### **현재 하고 있는 것**
회사에서는 ML과 관련있는 일을 하고있진 않습니다. SW이슈분석 및 레포팅을 하고 있습니다.

### **취미**
음악을 많이 듣습니다. 일할 때도 듣고 쉴때도 듣습니다. ☺️

**Comment by An9ieKim** [comment]
[[김은지](https://kr.linkedin.com/in/eunji-kim-37422618b/ko?trk=people-guest_people_search-card)]
프로젝트명 : 캐글 정복하기

### **인삿말**
안녕하세요. 김은지입니다. 반갑습니다. 앞으로 함께 공부하며 서로 성장해 나갈 수 있는 시간이 되길 바래봅니다! :)
 
### **간단한 이력**
산업공학과 학부 졸업 후 외국계 의류회사 SCM부서에서  약 6년간 데이터 분석가로 재직하였고,
퇴사 후 인공지능학과 석사 과정 진학하여 올해 초 졸업하였습니다.
현재 물류회사에서 데이터분석과 최적화 ML/DL 기술 연구를 수행하고 있습니다. 

### **현재 관심사**
그래프 모델, 추천 시스템에 관심을 가지고 공부해왔고, 앞으로 강화학습과 시계열 분석 기술들을 보다 깊게 공부해보고 싶습니다.

### **특별한 경험**

6년째 꾸준히 독서모임에 참여하고 있습니다. 같은 책을 읽고 다른 생각을 나누며 시야를 넓혀가는 일을 무척이나 좋아합니다. 

### **현재 하고 있는 것**
CJ 대한통운 물류기술연구소에서 데이터 분석 및 최적화 알고리즘 개발 연구원으로 일하고 있습니다.

### **취미**
달리기, 등산, 여행 같은 움직이는 활동과, 책 읽고 사색하는 정적인 활동 모두 다 좋아합니다!

**Comment by kcyoon689** [comment]
[김채윤](https://www.linkedin.com/in/chaeyoon-kim689/)
프로젝트명 : 캐글 정복하기

인삿말
안녕하세요! AI Researcher로 일하고 있는 김채윤입니다!

간단한 이력
인공지능을 전공으로 active learning, medical AI 분야로 석사를 졸업하고 
현재는 AI 스타트업에서 domain adaptation 등 비전 모델 관련된 연구를 하고 있습니다.

현재 관심사
해외 이직... 해외 살이.. 언젠가 해외 나가는 것을 최대 관심사로 두고 있습니다.

취미
귀여운 동물들 사진 감상, 오목, 테트리스..(테트리스는 나름 조금 잘하는 것 같아요…같이 하실 분 환영..ㅎㅎ)

**Comment by HyunZ118** [comment]
프로젝트명 : 캐글 정복하기

인삿말
안녕하세요! 전현지입니다. 열심히 스터디 하면서 함께 성장했으면 좋겠습니다. 잘 부탁드립니다! 😊

간단한 이력
헬스케어 스타트업에서 근무한 경험이 있고, Medical domain에 관심이 많아 대학원 진학을 하게 되었습니다.
의료빅데이터 관련 연구 경험이 있으며, 현재는 영상을 주로 다루는 연구실에 소속되어 있습니다.

현재 관심사
AI 실력을 차근차근 탄탄하게 쌓기 위해 열심히 공부하며 많은 노력을 기울이고 있습니다.
Radiology, Pathology 분야를 좀 더 깊게 공부해보고 싶습니다.

특별한 경험
롯데월드 아쿠아리움에서 근무한 경험이 있습니다 ㅎㅎ 🐬 🐙 🐳 🐠

현재 하고 있는 것
Medical AI 연구실에서 대학원 진학을 앞두고 있습니다.

취미
음악 듣는 걸 너무너무 좋아하고(특히 밴드 음악), 아침 수영을 종종 하고 있습니다.

**Comment by bongjooncha** [comment]
[차봉준] <a></a><a href="https://github.com/bongjooncha" target="_blank">(github)</a></br>
프로젝트명 :  캐글 정복하기

<h3>인삿말</h3>
안녕하세요. 개발 공부를 하고 있는 차봉준 입니다.</br>

<h3>간단한 이력</h3>
올해 2월 졸업을 했고 현재 취업 준비 중입니다. 토목공학을 전공하고 복수전공으로 경영학을 전공했으나, 개발이 재미있어 개발자로 커리어 준비를 하고 있습니다.</br>

<h3>현재 관심사</h3>
블록체인 기술, 코인의 탄생 배경, 크립토 시장 내 금융 상품에 관심이 생겨 공부하고 있습니다.</br>

<h3>특별한 경험</h3>
올해 약 2주간 한국에 있는 도시 탐방을 하며 디지털 노마드 생활 체험을 해보았습니다.</br>

<h3>현재 하고 있는 것</h3>
취업 전 주식 및 코인 자동 매매 프로그램을 만드는 것을 목표로 열심히 만들고 있습니다.</br>

<h3>취미</h3>
개발 이외에도 무언가 제작하는 것이 취미입니다.
